### PR TITLE
Update psycopg2 to 2.8.6 (to support Python 3.9)

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by vsoch** [comment]
Ah yes Django 4.0 came out so we will need to pin to under that for now! Also please add a note to CHANGELOG.md and bump VERSION.

**Comment by iarspider** [comment]
I'm not sure I understand your question. 
1. Inside anaconda environment, Python version is 3.9.x, as seen from this line: https://gist.github.com/iarspider/9fcf3fba03b308b7ff5ed12863781f57#file-gistfile1-txt-L2
2. There is no `cp39` wheel for psycopg2-binary version 2.8.5, as seen here: https://pypi.org/project/psycopg2-binary/2.8.5/#files . Python 3.9 wheels were added in 2.8.6.
3. In [your build](https://github.com/spack/spack-monitor/runs/4156272314?check_suite_focus=true#step:3:1297) conda installs python 3.8, for which there is [a wheel](https://github.com/spack/spack-monitor/runs/4156272314?check_suite_focus=true#step:3:1496), and the installation succeeded.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,4 +14,5 @@ represented by the pull requests that fixed them. Critical items to know are:
 ## [master](https://github.com/spack/spack-monitor/tree/main) (master)
  - skeleton release (0.0.0)
  - first version that has specs before September 2021 (1.0.0)
+ - update psycopg2-binary to 2.8.6, put upper limit on Django version (1.1.2)
```
Comment (vsoch) [inline CHANGELOG.md:17]: I think we need this to be 2.0.1?

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-1.1.1
+1.1.2
```
Comment (vsoch) [inline VERSION:1]: This seems odd - what branch are you working off of?

Comment (vsoch) [reply VERSION:1]: Actually this is my mistake I didn't bump to 2.0.0! The version file should have that.

**Comment by vsoch** [comment]
okay should be fixed now - sorry about that!

**Review by vsoch — APPROVED** [review]

**Pull request merged**